doc/advanced-configuration/outgoing-connections.md

```diff
@@ -483,113 +264,141 @@ Run the following function in the MongooseIM shell to verify that the connection
-MongooseIM will start as many `rabbit` pools as XMPP hosts are served by
-the server.
+## RabbitMQ options
 
 The `Tag` parameter must be set to `event_pusher` in order to be able to use
 the pool for [`mod_event_pusher_rabbit`](../modules/mod_event_pusher_rabbit.md).
-Any other `Tag` can be used for any other RabbitMQ connection pool.
-
-The `ConnectionOptions` list can take following parameters as `{key, value`} pairs:
-
-* **amqp_host** (default: `"localhost"`) - Defines RabbitMQ server host (domain or IP address; both as a string);
-* **amqp_port** (default: `5672`) - Defines RabbitMQ server AMQP port;
-* **amqp_username** (default: `"guest"`) - Defines RabbitMQ server username;
-* **amqp_password** (default: `"guest"`) - Defines RabbitMQ server password;
-* **confirms_enabled** (default: `false`) - Enables/disables one-to-one publishers confirms;
-* **max_worker_queue_len** (default: `1000`; use `infinity` to disable it) -
-Sets a limit of messages in a worker's mailbox above which the worker starts
-dropping the messages. If a worker message queue length reaches the limit,
-messages from the head of the queue are dropped until the queue length is again
-below the limit.
-
-### Example
-
-```erlang
-{ougtoing_pools, [
- {rabbit, host, event_pusher, [{workers, 20}],
-  [{amqp_host, "localhost"},
-   {amqp_port, 5672},
-   {amqp_username, "guest"},
-   {amqp_password, "guest"},
-   {confirms_enabled, true},
-   {max_worker_queue_len, 100}]}
-]}.
-```
-
-## LDAP connection setup
+Any other `Tag` can be used for other purposes.
+
+#### `outgoing_pools.rabbit.*.connection.amqp_host`
+* **Syntax:** string
+* **Default:** `"localhost"`
+* **Example:** `amqp_host = "anotherhost"`
+
+#### `outgoing_pools.rabbit.*.connection.amqp_port`
+* **Syntax:** integer
+* **Default:** `5672`
+* **Example:** `amqp_port = 4561`
+
+#### `outgoing_pools.rabbit.*.connection.amqp_username`
+* **Syntax:** string
+* **Default:** `"guest"`
+* **Example:** `amqp_username = "corpop"`
+
+#### `outgoing_pools.rabbit.*.connection.amqp_password`
+* **Syntax:** string
+* **Default:** `"guest"`
+* **Example:** `amqp_password = "guest"`
+
+#### `outgoing_pools.rabbit.*.connection.confirms_enabled`
+* **Syntax:** boolean
+* **Default:** `false`
+* **Example:** `confirms_enabled = false`
+* **Description:** Enables/disables one-to-one publishers confirms.
+
+#### `outgoing_pools.rabbit.*.connection.max_worker_queue_len`
+* **Syntax:** non-negative integer or `"infinity"`
+* **Default:** `1000`
+* **Example:** `max_worker_queue_len = "infinity"`
+* **Description:** Sets a limit of messages in a worker's mailbox above which the worker starts dropping the messages. If a worker message queue length reaches the limit, messages from the head of the queue are dropped until the queue length is again below the limit. Use `infinity` to disable.
+
+## LDAP options
+
+#### `outgoing_pools.ldap.*.connection.servers`
+* **Syntax:** a list of strings
```

Comment:
minor: array of strings, but I understand that 'list' might be more understandable to the user, however so far in the docs it's been 'array'